### DISCUSSION TITLE
Select multiple and searchable field in Frontend Blade FIle

### DISCUSSION THREAD

**Discussion by fr3deric** [filamentphp/filament#11125] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.14

### How can we help you?

Hey all, 

I am currently using Laravel Jestream (with the Livewrie + Blade stack) to render the frontend of my website. As the "backend" I am using Filament v3.2.14. I've created some DB-Tables and Forms in the admin-backend and that all is working fine. I now wanted to build a form in the freontend. Tabs are showing fine, the 'select' works as long as I do not use any javascript.  If I am using javasciprt, e.g. with `searchable` or `native(false)`  I only see an empty select box. 

I exactly followed the tutorial "Adding a form to a Livewire component" . But I cannot figure out what's causing the behaviour. There are no errors in the browser's console as well no 404's when I look at the network. 

Any help is appreciated,
thanks in advance,

Frederic

**Comment by ohiader** [comment, marked as answer]
You need to add `@filamentStyles` and `@filamentScripts` to your layout file. I used [stacks](https://laravel.com/docs/11.x/blade#stacks) to only add them to the pages that need the resources.

[It's in the forms installation docs](https://filamentphp.com/docs/3.x/forms/installation#configuring-your-layout)

**Reply by fr3deric** [reply]
Thank you so much. That was what I was missing.
